### ISSUE TITLE
"Getting Started" tutorial doesn't work 

### ISSUE THREAD

**Issue by andrewedstrom** [concourse/concourse#2496]
# Bug Report

When we try to follow the [Getting Started documentation](https://concourse-ci.org/getting-started.html), our concourse fails to start with the following error:

```
→ wget https://concourse-ci.org/docker-compose.yml
→ docker-compose up
local-concourse_concourse-db_1 is up-to-date
Recreating local-concourse_concourse_1 ... done
Attaching to local-concourse_concourse-db_1, local-concourse_concourse_1
concourse-db_1  | The files belonging to this database system will be owned by user "postgres".
concourse-db_1  | This user must also own the server process.
concourse-db_1  |
concourse-db_1  | The database cluster will be initialized with locale "en_US.utf8".
concourse-db_1  | The default database encoding has accordingly been set to "UTF8".
concourse-db_1  | The default text search configuration will be set to "english".
concourse-db_1  |
concourse-db_1  | Data page checksums are disabled.
concourse-db_1  |
concourse-db_1  | fixing permissions on existing directory /database ... ok
concourse-db_1  | creating subdirectories ... ok
concourse-db_1  | selecting default max_connections ... 100
concourse-db_1  | selecting default shared_buffers ... 128MB
concourse-db_1  | selecting dynamic shared memory implementation ... posix
concourse-db_1  | creating configuration files ... ok
concourse-db_1  | running bootstrap script ... ok
concourse-db_1  | performing post-bootstrap initialization ... ok
concourse-db_1  |
concourse-db_1  | WARNING: enabling "trust" authentication for local connections
concourse-db_1  | You can change this by editing pg_hba.conf or using the option -A, or
concourse-db_1  | --auth-local and --auth-host, the next time you run initdb.
concourse-db_1  | syncing data to disk ... ok
concourse-db_1  |
concourse-db_1  | Success. You can now start the database server using:
concourse-db_1  |
concourse-db_1  |     pg_ctl -D /database -l logfile start
concourse-db_1  |
concourse-db_1  | waiting for server to start....2018-08-16 16:31:32.115 UTC [42] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
concourse-db_1  | 2018-08-16 16:31:32.128 UTC [43] LOG:  database system was shut down at 2018-08-16 16:31:31 UTC
concourse-db_1  | 2018-08-16 16:31:32.134 UTC [42] LOG:  database system is ready to accept connections
concourse-db_1  |  done
concourse-db_1  | server started
concourse-db_1  | CREATE DATABASE
concourse-db_1  |
concourse-db_1  | CREATE ROLE
concourse-db_1  |
concourse-db_1  |
concourse-db_1  | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
concourse-db_1  |
concourse-db_1  | waiting for server to shut down....2018-08-16 16:31:32.670 UTC [42] LOG:  received fast shutdown request
concourse-db_1  | 2018-08-16 16:31:32.672 UTC [42] LOG:  aborting any active transactions
concourse-db_1  | 2018-08-16 16:31:32.676 UTC [42] LOG:  worker process: logical replication launcher (PID 49) exited with exit code 1
concourse-db_1  | 2018-08-16 16:31:32.676 UTC [44] LOG:  shutting down
concourse-db_1  | 2018-08-16 16:31:32.688 UTC [42] LOG:  database system is shut down
concourse-db_1  |  done
concourse-db_1  | server stopped
concourse-db_1  |
concourse-db_1  | PostgreSQL init process complete; ready for start up.
concourse-db_1  |
concourse-db_1  | 2018-08-16 16:31:32.782 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
concourse-db_1  | 2018-08-16 16:31:32.782 UTC [1] LOG:  listening on IPv6 address "::", port 5432
concourse-db_1  | 2018-08-16 16:31:32.784 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
concourse-db_1  | 2018-08-16 16:31:32.794 UTC [69] LOG:  database system was shut down at 2018-08-16 16:31:32 UTC
concourse-db_1  | 2018-08-16 16:31:32.798 UTC [1] LOG:  database system is ready to accept connections
concourse_1     | 1 error occurred:
concourse_1     |
concourse_1     | * must configure basic auth, OAuth, UAAAuth, or provide no-auth flag
local-concourse_concourse_1 exited with code 1
```

I see this as well if I just try to run the `quickstart` command on the concourse/concourse docker image
```
→ docker run -it concourse/concourse quickstart
1 error occurred:

* must configure basic auth, OAuth, UAAAuth, or provide no-auth flag
```

Also it's worth noting that that error is not helpful. It suggests there is a flag called `--no-auth`. The actual flag to disable auth is `--no-really-i-dont-want-any-auth`.

The following can also be handy:

* Concourse version: 3.13.0
* Deployment type (BOSH/Docker/binary): docker-compose
* Did this used to work? I assume so.

**Comment by vito** [comment]
Works fine for me. Why is your Concourse version 3.13.0? Did you have it fetched locally still? That file is meant for 4.0+ (whatever `latest` is). I guess we should tag the image in the .yml file.

**Comment by jama-pivotal** [comment]
It might be because Pivotal "official" supported concourse is still 3.13.0 (because reasons)

**Comment by jama-pivotal** [comment]
Closing since it doesn't seem to be reproducible.

**Comment by ruccoon** [comment]
I got the same error while launching the quick start container

concourse_1     | * must configure basic auth, OAuth, UAAAuth, or provide no-auth flag
concourse_concourse_1 exited with code 1

`$ docker logs 3862446cf229
{"timestamp":"2019-03-06T22:20:52.436043800Z","level":"info","source":"atc","message":"atc.cmd.start","data":{"session":"1"}}
default team auth not configured: ErrAuthNotConfiguredFromFlags
{"timestamp":"2019-03-06T22:20:53.461531200Z","level":"info","source":"atc","message":"atc.cmd.finish","data":{"duration":128200,"session":"1"}}
`

**Comment by chenbh** [comment]
hey @ruccoon we forgot to update the `docker-compose.yml` for the recently released 5.0.0, we have submitted a PR at https://github.com/concourse/docs/pull/16, you can take a look at it for a work-around
